### PR TITLE
Release v0.9.1

### PR DESCRIPTION
**Pull request by lentzi90**
**What this PR does / why we need it**: This adds a patch to v0.9 in order to fix the unit tests.

**Which issue this PR fixes** *(use the format `fixes #<issue number>(, fixes #<issue_number>, ...)` to automatically close the issue when PR gets merged)*: -

**Public facing documentation PR** *(if applicable)*

**Special notes for reviewer**:

**Checklist:**

- [ ] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/ck8s-cluster/blob/master/WIP-CHANGELOG.md)
- [x] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [x] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.

### CODE REVIEW

File: client/cluster_test.go [hunk 1]
```diff
@@ -124,7 +124,7 @@ func TestAddMachine(t *testing.T) {
 
 	latestImageMaster := api.LatestImage(cloudProvider, api.Master)
 	latestImageWorker := api.LatestImage(cloudProvider, api.Master)
-	firstImageMaster := cloudProvider.MachineImages(api.Master)[0]
+	firstImageMaster := cloudProvider.MachineImages(api.Master)[2]
```
Comment (Xartos) [inline client/cluster_test.go]: Does this needs to be latest image - 1? If so would it make sense to do:
```suggestion
  allImages = cloudProvider.MachineImages(api.Master)
  firstImageMaster := allImages[ len(allImages) - 2 ]
```

Comment (lentzi90) [reply client/cluster_test.go]: Yes that looks much better! I guess it doesn't *have* to be latest - 1 if there are multiple with the same minor kubernetes version, but the point of the test (I think) is just to see that something other than latest also works.

File: client/cluster_test.go [hunk 2]
```diff
@@ -124,7 +124,8 @@ func TestAddMachine(t *testing.T) {
 
 	latestImageMaster := api.LatestImage(cloudProvider, api.Master)
 	latestImageWorker := api.LatestImage(cloudProvider, api.Master)
-	firstImageMaster := cloudProvider.MachineImages(api.Master)[0]
+  allImages = cloudProvider.MachineImages(api.Master)
+  firstImageMaster := allImages[ len(allImages) - 2 ]
```
Comment (viktor-f) [inline client/cluster_test.go]: To have the same indentation as the other lines.
```suggestion
      allImages = cloudProvider.MachineImages(api.Master)
      firstImageMaster := allImages[ len(allImages) - 2 ]
```

**Review by viktor-f — COMMENTED** [review]
One minor comment otherwise it looks good!

File: client/cluster_test.go (hunk 2 above)
Comment (lentzi90) [reply client/cluster_test.go]: Oops... should have checked tabs vs space :smile:

**Review by viktor-f — APPROVED** [review]

**Review by Xartos — APPROVED** [review]
LGTM!

**Pull request merged**